P33J3/CMS-NuKeys
# Chore: Added barrel pattern

###  What does this PR do?

This PR adds the barrel pattern to the customized NodeJS Scaffold

###  Description of Task to be completed?

The overall task is to import and export all modules used from relevant index files.

### How should this be manually tested?

This should be manually tested by running the server to ensure the routes can be accessed and in turn access the MVC model used.

###  Any background context you want to provide?

N/A

###  What are the relevant pivotal tracker stories?

To be added

### Questions:

1. Are we using the trello board as our pivotal tracker?
3. I am having issues following the tutorial provided for creating the barrel pattern. I resorted to 'export default' in order to successfully create the barrel pattern. Could you illustrate where I am going wrong please? (Screenshot provided of error encountered)

### Screenshots (if appropriate)

![barrel export error](https://user-images.githubusercontent.com/81102471/165660997-384d71c4-1e1e-4d2a-8d48-a1f2d0d62b9c.jpg)
